packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -168,9 +169,28 @@ export default class AgoraIpOrderCtrl {
     );
   }
 
-  loadIpOffers() {
+  manageLoadIpOffers() {
     this.loading.ipOffers = true;
+    this.ipOffers = [];
 
+    if (
+      get(this.model, 'selectedService.type') ===
+      PRODUCT_TYPES.dedicatedServer.typeName
```

Comment:
Could we get rid of lodash dependencies and use native js instead ?
```suggestion
this.model?.selectedService?.type === PRODUCT_TYPES.dedicatedServer.typeName
```